.github/workflows/ci.yml

```diff
@@ -94,6 +94,20 @@ jobs:
           platform: ${{matrix.platform}}
           config: ${{matrix.config}}
           build-llvm: true
+      - name: Build Slang for Wasm
```

Comment:
We don't need to run this build for every platform matrix.
Instead, wasm should be one of the entries in `matrix.platform`, and this secion should be what "Build Slang" step is doing for that `wasm` platform.

Reply:
I see.
But `wasm` is an actual platform. Do we want to use Windows for that?
I can make wasm build for Windows Release and Debug if that's what you meant.

Reply:
Let's use linux to build it.

Reply:
And let's only do a release build.

Reply:
Ok. Linux Release it is, then.